### ISSUE TITLE
Can you explain the code in this project?

### ISSUE THREAD

**Issue by zhangqianhui** [carpedm20/DCGAN-tensorflow#63]
`if np.mod(counter, 100) == 1:
                    if config.dataset == 'mnist':
                        samples, d_loss, g_loss = self.sess.run(
                            [self.sampler, self.d_loss, self.g_loss],
                            feed_dict={self.z: sample_z, self.images: sample_images, self.y:batch_labels}
                        )
                    else:`

when sampling , the feeding of y is batch_labels for mnist training?
I think should be sample_labels?

**Comment by carpedm20** [comment]
I recommend you to read the [paper](https://arxiv.org/abs/1511.06434) Section 8. They trained DCGAN conditioned on the label for MNIST dataset to see the result of conditional generative models.
